VATSpy.dat

```diff
@@ -18259,6 +18263,12 @@ UMMV|Minsk|UMMM|UMMV
 UNKL|Krasnoyarsk||UNKL
 UNNT|Novosibirsk||UNNT
 URRV|Rostov||URRV
+URRV-A|Rostov ACC - Astrakhan|URRV_A|URRV-A
+URRV-K|Rostov ACC - Krasnodar|URRV_K|URRV-K
+URRV-R|Rostov ACC - Platov|URRV_R|URRV-R
+URRV-S|Rostov ACC - Sochi|URRV_S|URRV-S
+URRV-V|Rostov ACC - Vladikavkaz|URRV_V|URRV-V
+URRV-W|Rostov ACC - Volgograd|URRV_W|URRV-W

```

Comment:
Same as above.

![image](https://user-images.githubusercontent.com/35482228/200086445-02fe7e41-c7c9-4f9b-ac9b-aa74e5346020.png)

Reply:
@Adrian2k These sectors was added to our sector file 1 month ago.  We will connect on separate sectors not only on events. To connect on them, we need Vatspy, because pilots don’t understand that separate sector doesn’t cover whole FIR.

Reply:
I agree with you about UGGG and UBBA. We don’t really need it. But we need URRV.

Reply:
@Adrian2k @NelisV